covasim/webapp/index.html

```diff
@@ -36,199 +36,222 @@
 
     <!-- About modal -->
     <b-modal id="about" ok-only ok-variant="secondary" ok-title="Close">
       <template v-slot:modal-title>About {{ app.title }}</template>
       <p>{{ app.title }} is an agent-based model that simulates the transmission of COVID-19 (novel coronavirus, SARS-CoV-2) in a population. Each individual in the model may pass through the following stages of COVID-19 infection: susceptible, exposed, infectious, and recovered (SEIR). The epidemiological parameter values are taken from literature.</p>
-      <p>Viral transmission occurs on a fixed contact network with undirected edges. Each day, infectious individuals expose susceptible individuals in neighboring nodes to possible infection. By default, the daily probability of infection, number of contacts, and mean duration of infectiousness roughly equates to R0 = 2.05 and a doubling time of 6-7 days. For more information, please visit the <a href="{{ app.github_url }}" target="_blank">GitHub</a> repository.</p>
-      <p>Note: Models are only as good as the values of the parameters put into them. This is a rapidly changing situation, and we will update content as relevant information is received.</p>
+      <p>Viral transmission occurs on a fixed contact network with undirected edges. Each day, infectious individuals expose susceptible individuals to possible infection. By default, the daily probability of infection, number of contacts, and mean duration of infectiousness roughly equates to R0 = 2.5 and a doubling time of 6-7 days. For more information, please visit the <a href="https://github.com/InstituteforDiseaseModeling/covasim" target="_blank">GitHub</a> repository.</p>
+      <p><b>Note:</b> Models are only as good as the values of the parameters put into them. This is a rapidly changing situation, and we will update content as relevant information is received.</p>
+      <p><b>Acknowledgements:</b> Covasim is currently under active development by IDM's COVID-19 Response Team. Model development is led by Daniel Klein, Cliff Kerr, Robyn Stuart, Romesh Abeysuriya, and Dina Mistry. UI design is led by Lauren George, William Chen, and Scott Ayers, inspired by a prototype from Brian Lovin. For the full list of contributors, please see our <a href="https://github.com/InstituteforDiseaseModeling/covasim/graphs/contributors" target="_blank">Contributors</a> page</a>.</p>
     </b-modal>
 
 
   </header>
 
   <div class="content">
 
-    <b-sidebar id="parameters" :visible="panel_open" no-header bg-variant="white" :width="panel_width+'vw'" shadow>
+    <b-sidebar id="parameters" :visible="panel_open" no-header class="sidebar" :width="panel_width+'vw'" shadow>
       <div class="px-3">
         <header class="d-flex justify-content-between">
-          <h4 class="my-2">Campaign Builder</h4>
-          <button class="close" @click="close_panel" aria-label="Close">
-            <span aria-hidden="true">&times;</span>
+          <h4 class="my-2">Simulation configuration</h4>
```

Comment:
Sentence casing is hard to parse when it's a header. I think you should revert all of these changes.

Reply:
So I was wondering why I'm averse to title case. TIL, it's an Americanism:

https://www.herosmyth.com/article/headline-capitalization-ux-title-case-vs-sentence-case

"If your audience is only in the US, then reference other factors to determine which case to implement in your headlines.

Otherwise, if you’re writing internationally-targeted content, think global—use sentence case to cater to more audiences rather than fewer."

So strong vote from this Aussie to keep sentence case ;)

Reply:
Fair enough. I'm an American working in an American org. My bias is showing :).